spec/models/book_spec.rb

```diff
@@ -10,4 +11,7 @@
   it { is_expected.to validate_presence_of(:title) }
   it { is_expected.to validate_presence_of(:publisher) }
   it { is_expected.to validate_presence_of(:year) }
+
+  it { is_expected.to have_many(:rents).class_name('Rent') }
```

Comment:
when things are standard, you don't need to specify :) You can remove the `class_name('Rent')`